### PR TITLE
[12.0] [MIG] purchase_order_weight

### PR DESCRIPTION
**Pull request by vvrossem**
#### `display_weight` and `display_unit`
These fields appear but are not editable as they are `related` fields therefore read only by default.
purchase_order_weight1

#### Request for quotation (pdf)
purchase_order_weight2

#### Purchase order confirmation (pdf)
purchase_order_weight3

### CODE REVIEW

File: purchase_order_weight/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (robinkeunen) [inline purchase_order_weight/__init__.py]: missing EOL

File: purchase_order_weight/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+##############################################################################
+{
+    "name": "Purchase Order Weight",
+    "version": "12.0.1.0.0",
+    "depends": [
+        'base',
+        'purchase',
+        'beesdoo_product',
+    ],
+    "author": "Coop IT Easy - Robin Keunen <robin@coopiteasy.be>",
```
Comment (robinkeunen) [inline purchase_order_weight/__manifest__.py]: Coop IT Easy SCRLfs

**Review by robinkeunen — CHANGES_REQUESTED** [review]
Yo,

Good stuff, but I think it'd be better to remove the dependency on `beesdoo_product.`

A little black on this module would be nice as well.

File: purchase_order_weight/__init__.py (hunk 1 above)
Comment (vvrossem) [reply purchase_order_weight/__init__.py]: done :)

File: purchase_order_weight/__manifest__.py (hunk 2 above)
Comment (vvrossem) [reply purchase_order_weight/__manifest__.py]: done :)

**Comment by vvrossem** [comment]
@robinkeunen thanks for the fixup! training

**Comment by vvrossem** [comment]
@robinkeunen ok to merge?

**Review by robinkeunen — APPROVED** [review]

**Comment by robinkeunen** [comment]
@vvrossem yay

**Pull request merged**